_includes/agenda.njk

```diff
@@ -1,4 +1,4 @@
-<ol class="schedule | tbds-inline-stack  tbds-inline-stack--gap-4">
+<ol class="schedule | tbds-inline-stack  tbds-inline-stack--gap-4 container">
```

Comment:
I think that in future we need to look at how to avoid content getting cut off horizontally. Trello, for example add scroll bars to content that can scroll to avoid people not realising that there are more columns to the right. I think whether we add `container` here or not content gets cut-off at particular screen sizes. I'll add an issue.

Reply:
Sounds good. The schedules do have a scroll bar at the bottom. Is the bar not visible enough and we should customize it a little, or are you thinking about a different type of solution down the road?

Reply:
> Sounds good. The schedules do have a scroll bar at the bottom. Is the bar not visible enough and we should customize it a little, or are you thinking about a different type of solution down the road?

Yeah, I think it's another issue for later.